packages/stories-block/src/block/block-types/latest-stories/edit.js

```diff
@@ -70,7 +70,7 @@ function LatestStoriesEdit({ attributes, setAttributes }) {
       const stories = await apiFetch({
         path: addQueryArgs(storiesApi, {
           per_page: 20,
-          _embed: 'author',
+          _embed: 'author,wp:featuredmedia',
```

Comment:
Unrelated fix

Reply:
Curious, what was broken / what does this fix?

Reply:
The poster image was not displaying as it was not loaded in the API.